packages/data-sdk/src/devices/Device.ts

```diff
@@ -565,7 +565,7 @@ export class Device extends BaseDevice {
     return await createShareLink(share, view);
   }
 
-  async createDevice(device: IDevice): Promise<IDevice> {
+  static async createDevice(device: IDevice): Promise<IDevice> {
```

Comment:
you can definitely still pull these into those little _api_ functions like the others, and just "export" them statically on `Device`. I think there _are_ some good advantages (at least as we work through some of the "responsibilities" of these singletons/classes and refactor them) to think of them that way.

Reply:
yeah I saw the comment after I push the changes, but I just reverted to have them as single files per function and just import them in the class